### PR TITLE
Fix lock state icon position in detailed files' lists

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
**Before :** 
Capture d’écran du 2023-10-11 15-38-40
**After :**
Capture d’écran du 2023-10-11 15-38-23

### CODE REVIEW

**Review by jancborchardt — COMMENTED** [review]
Looks good, butnis it not better to horizontally center it? And also, will this work with the Vue rewrite @skjnldsv?

**Comment by juliushaertl** [comment]
@jancborchardt No, see discussion in https://github.com/nextcloud/files_lock/issues/154

**Comment by jancborchardt** [comment]
@juliushaertl the should this fix be merged so it's fixed for pre-Files2Vue versions?

**Comment by juliushaertl** [comment]
Need to double check all compatible versions first, but yes we should merge before f2v for the stable releases

File: css/files_lock.css [hunk 1]
```diff
@@ -4,7 +4,7 @@ tr[data-locked='true'] .fileactions .action.action-menu.permanent {
 }
 .locking-inline-state.icon-password {
 	padding: 17px 23px;
-	margin-right: -45px;
+	margin-right: -49px;
```
Comment (juliushaertl) [inline css/files_lock.css:7]: Unfortunately this has a similar issue with entries that are not shared:

Screenshot 2023-11-06 at 09 16 17

The following style changes seem to work though:
Screenshot 2023-11-06 at 09 15 52

**Review by juliushaertl — CHANGES_REQUESTED** [review]
Changes requested inline

File: css/files_lock.css (hunk 1 above)
Comment (Jerome-Herbinet) [reply css/files_lock.css]: Check last commit. Both UI content (shared and not shared) should display lock properly, including in grid view.

**Review by juliushaertl — APPROVED** [review]
Thanks 👍 Tested and works

**Pull request merged**